dev/dev-env/template-scenarios.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs, lib, scenarios, nix-bitcoin }:
+with lib;
+rec {
+  # For more examples, see `scenarios` and `exampleScenarios` in ./src/test/tests.nix
```

Comment:
Also see `dev-scenarios`?

In general, I was quite confused about the various locations of the scenarios. We have
- example scenarios, which are somewhat documenting what's happening in tests.nix
- we have `template-scenarios.nix` (aka scenarios.nix), which is a file supposed to be changed by a dev user
- we have `dev-scenarios.nix`, which are our collection of scenarios for users (although the line to `dev-features.sh` is quite blurry)

It seems to all make sense, but is still quite confusing. I wonder if there was a way to somehow simplify this.

Reply:
A user checking the definition of `scenarios` (tests.nix) will see that it includes `dev-scenarios.nix`.

- `template-scenarios.nix` must be a separate file because it's part of the user's dev env.
- We need `dev-scenarios.nix` because defining (and editing) complex scenarios inline as a bash string in `dev-features.sh` is bad UX.
   We could move `dev-scenarios.nix` into `tests.nix`, but this adds unneeded complexity to `tests.nix`. I don't see how we could improve the file layout.

Reply:
maybe mention dev-scenarios.nix and its purpose in the dev readme?